.gitignore

```diff
@@ -113,3 +113,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#for vscode
+.vscode
```

Comment:
Configure your editor
1. To add a new line at the end of the file while saving.
2. Trim any extra lines at the end of files.
3. Trim any trailing whitespace.

Reply:
Done all three.